app/src/main/java/com/sopt/now/compose/LoginActivity.kt

```diff
@@ -0,0 +1,163 @@
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.sopt.now.compose.ui.theme.NOWSOPTAndroidTheme
+
+@Suppress("DEPRECATION")
+class LoginActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            NOWSOPTAndroidTheme {
+                Surface(
+                    modifier = Modifier.fillMaxSize(),
+                    color = MaterialTheme.colorScheme.background
+                ) {
+                    Login()
+                    val userInfo = intent.getSerializableExtra("user_info") as? UserInfo
+                    userInfo?.let { user ->
+                        Login(user)
+                    }
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun Login(userInfo: UserInfo?=null) {
+    var id by remember { mutableStateOf("") }
+    var password by remember { mutableStateOf("") }
+
+    val context = LocalContext.current
+
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(30.dp),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Spacer(modifier = Modifier.height(40.dp)) //위에서부터 높이
+
+        Text(
+            text = "Welcome to SOPT",
+            fontSize = 30.sp,
+            fontWeight = FontWeight.Bold,
+//            textAlign = TextAlign.Center,
+            modifier = Modifier.align(Alignment.CenterHorizontally),
+        )
+        Spacer(modifier = Modifier.height(50.dp))
+
+        Text(
+            text = "ID",
+            fontSize = 20.sp,
+            fontWeight = FontWeight.Bold,
+            textAlign = TextAlign.Start,
+            modifier = Modifier.fillMaxWidth()
+        )
+        Spacer(modifier = Modifier.height(30.dp))
+        TextField(
+            value = id,
+            onValueChange = { id = it },
+            placeholder = { Text("아이디를 입력해주세요") },
+            modifier = Modifier.fillMaxWidth(),
+            keyboardOptions = KeyboardOptions.Default.copy(imeAction = ImeAction.Next),
+        )
+        Spacer(modifier = Modifier.height(40.dp))
+        Text(
+            text = "비밀번호",
+            fontSize = 20.sp,
+            fontWeight = FontWeight.Bold,
+            textAlign = TextAlign.Start,
+            modifier = Modifier.fillMaxWidth()
+        )
+        Spacer(modifier = Modifier.height(30.dp))
+        TextField(
+            value = password,
+            onValueChange = { password = it },
+            placeholder = { Text("비밀번호를 입력해주세요") },
+            modifier = Modifier.fillMaxWidth(),
+            visualTransformation = PasswordVisualTransformation(),
+            keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Password),
+        )
+        Spacer(modifier = Modifier.height(200.dp))
```

Comment:
```suggestion
        Spacer(modifier = Modifier.height(1f))
```

이렇게 쓰면 남은 공간을 다 채워줍니당!